packages/bpk-component-inset-banner/src/BpkInsetBanner.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkInsetBanner.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  title?: string;
+  subheadline?: string;
+  logo?: string;
+  backgroundColor: string;
+  variant: 'onLight' | 'onDark';
+  body?: {
+    text: string;
+    link?: string;
+    linkText?: string;
+  };
+  // [TO-DO] In future iteration we will add a button to CTA section with popover functionality
+  callToAction?: {
+    text?: string;
+  };
+};
+
+const BpkInsetBanner = ({
+  backgroundColor,
+  body,
+  callToAction,
+  logo,
+  subheadline,
+  title,
+  variant,
+}: Props) => {
+  const classNames = [];
+  if (variant === 'onLight') {
+    classNames.push('bpk-inset-banner--on-light');
+  }
+  if (variant === 'onDark') {
+    classNames.push('bpk-inset-banner--on-dark');
+  }
+  if (body) {
+    classNames.push('bpk-inset-banner--with-body');
+  }
+
+  return (
+    <div>
+      <div
+        className={getClassName('bpk-inset-banner', ...classNames)}
+        style={{
+          backgroundColor,
+        }}
+      >
+        <div className={getClassName('bpk-inset-banner--content-container')}>
+          {logo ? (
+            <img
+              className={getClassName('bpk-inset-banner--image')}
+              src={logo}
+              alt=""
```

Comment:
I think logos would need an alt looking at the figma designs, which would be a partner name

So it could be that we make logo an ReactElement and all this is passed into instead of building ourselves?

Reply:
The current usage in the ads is that the partner logo is hidden in the voice over and the partner info is passed to voice over via an aria-label as `Advertisement by {partner}` that is read out at the start of the advert. In this case we can pass some a11y text as a prop that can be added to the content container div?

Reply:
Hmm good question maybe this is something we need to confirm up with design and make sure that is reflected in Figma to be surrounding rather than just the logo.

I haven't tried it yet but what would the impact be for a screenreader if the whole thing is surrounded with the aria label and then there is more things to track inside it

Reply:
I just checked with Eileen and she confirmed we want to use the current method used in our ads which is to have the `Advertisement by {partner}` aria-label added to the container of the card, so that all the elements are contained in a group and can be skipped if the user doesn't want to listen to the content of the card.